go-controller/pkg/metrics/node.go

```diff
@@ -34,9 +34,20 @@ var metricOvnNodePortEnabled = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "Specifies if the node port is enabled on this node(1) or not(0).",
 })
 
+// metric to get the size of ovnkube.log file
+var metricOvnKubeNodeLogFileSize = prometheus.NewGaugeVec(prometheus.GaugeOpts{
+	Namespace: MetricOvnkubeNamespace,
+	Subsystem: MetricOvnkubeSubsystemNode,
+	Name:      "logfile_size",
```

Comment:
https://prometheus.io/docs/practices/naming/

`log_file_size_bytes` ?

Reply:
sure.

Reply:
@martinkennelly  I quickly looked in to Kubernetes source base, and looks like they also heavily use `logfile` everywhere. We also use 'logfile` as one word everywhere. So, I am going to use 'logfile.

Will change to logfile_size_bytes